modules/hp_timeline/config/install/field.storage.paragraph.field_timeline_title.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - paragraphs
+id: paragraph.field_timeline_title
+field_name: field_timeline_title
```

Comment:
should this be field_hp_timeline_text? Remember we want to prefix hp wherever possible to indicate that this field is controlled by howard paragraphs module.